jelixir/lib/jelixir.ex

```diff
@@ -20,6 +24,78 @@ defmodule JelixirLib do
     end
   end
 
+  def conver(task, file_name_string) when task == :schema do
+    with {:read_file_result, {:ok, file_content}} <-
+           {:read_file_result, read_file(file_name_string, ".ex")} do
+      if !File.exists?(@default_folder_out) do
+        File.mkdir(@default_folder_out)
+      end
+
+      parse_schema_result =
+        String.split(file_content, "\n")
+        |> Enum.map(fn line ->
+          line |> String.trim() |> read_line_schema()
+        end)
+        |> Enum.filter(fn item -> item != {:skip, nil} end)
+        |> Enum.into(%{})
+
+      module_name = parse_schema_result["module_name"]
+      schema_name = parse_schema_result["schema_name"]
+
+      node_result =
+        parse_schema_result
+        |> Map.delete("module_name")
+        |> Map.delete("schema_name")
+
+      create_gen(module_name, schema_name, node_result)
+    else
+      {:read_file_result, _} -> IO.inspect("Can't read file with name: #{file_name_string}")
+      {:name_result, _} -> IO.inspect("Can't parse file with name: #{file_name_string}")
+    end
+  end
+
+  def conver(_task, _file_name_string) do
+    {:error, "No thing to do"}
+  end
+
+  def read_line_schema(flat_line) do
+    cond do
+      String.contains?(flat_line, "defmodule") ->
+        module_name_result = Regex.run(@regex_pattern_module_name, flat_line)
+
+        if module_name_result do
+          module_name = Enum.at(module_name_result, 1)
+          {"module_name", module_name}
+        else
+          {"skip", nil}
+        end
+
+      String.contains?(flat_line, "schema") ->
```

Comment:
@ttpho 
I think it will be wrong when the field name contains `schema`. Example `field(:user_schema, :text)`
I think we should check by using regex pattern.

Here is my suggestion

```
def read_line_schema(flat_line) do
  cond do
    [_line, module_name] = Regex.run(@regex_pattern_module_name, flat_line) ->
      {"module_name", module_name}

    [_line, schema_name] = Regex.run(@regex_pattern_schema_name, flat_line) ->
      {"schema_name", schema_name}

    [_line, field_name, field_type] = Regex.run(@regex_pattern_field, flat_line) ->
      {field_name, field_type}

    true ->
      {:skip, nil}
  end
end
```

Reply:
@quangvo09 Thank you for your suggestion.

Reply:
@quangvo09 Your suggestion is good, but `Regex.run(@regex_pattern_module_name, flat_line)` will return is nil and not math with `[_line, module_name]`. So I think it is not the best solution.